assets/js/googlesitekit-dashboard-details.js

```diff
@@ -48,8 +51,21 @@ domReady( () => {
 	);
 
 	if ( renderTarget ) {
+		// Using global preloaded data since we cannot use selectors
+		// outside the Root component.
+		const isAuthenticated =
+			global._googlesitekitAPIFetchData.preloadedData[
+				'/google-site-kit/v1/core/user/data/authentication'
+			].body.authenticated;
```

Comment:
@aaemnnosttv sounds like a good alternative. No, we didn't consider it, this idea didn't come up when we brainstormed it.

Reply:
I'm definitely in favor of making the API request over reaching into the global.

Yet another alternative approach could be to avoid the lookup on client side entirely by using a `data-` attribute on the render target `div` that is set from the server-side. We already do that in a few other cases e.g. in case we need to render a module setup the `data-setup-module-slug` attribute is set. We could for example set a `boolean``data-view-only` attribute and if it is set (or set to `true`) we would know to use the other context.

I think that may be preferable as it would simplify/speed up the client-side check here.

Reply:
Sounds good @felixarntz , let's go with an added attribute instead so we don't need to make more substantial changes on the client 👍

Reply:
Good shout. I've made the changes and tested on my local.